### PR TITLE
[MODIN]: Add time results verifier for getting started notebook

### PR DESCRIPTION
**Pull request by dchigarev**
# Existing Sample Changes
## Description

[NOTE: This PR is a copy of the accidentally closed #843 (can't be reopened)]

The notebook is supposed to show the performance improvement after switching from stock Pandas to Modin, it compares the execution times of Pandas and Modin in various scenarios. The previous version of the notebook didn't imply that the Modin could be slower than Pandas anyhow and so confused users if such happened.
 
The changes in this PR introduce a time verifier, that compares Modin and Pandas execution times and prints a note if Modin appears to be slower with the possible reasons for it (poor CPU performance, old Modin version, silly environment variables).

Fixes [Jira MODIN6-135](https://jira.devtools.intel.com/browse/MODIN6-135).

## External Dependencies

The changes do not add any new dependencies.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Implement fixes for Jiras

## How Has This Been Tested?

- [x] Command Line

To test the changes locally:
1. Combine steps from the [`sample.json`](https://github.com/oneapi-src/oneAPI-samples/blob/master/AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/sample.json) into a single bash script.
<details><summary>script</summary>

```
set -e # Terminate the script on first error
source $(conda info --base)/etc/profile.d/conda.sh # Bypassing conda's disability to activate environments inside a bash script: https://github.com/conda/conda/issues/7980
conda create -y -n intel-aikit-modin intel-aikit-modin -c intel
conda activate intel-aikit-modin
pip install -r requirements.txt # Installing notebook's dependencies
pip install runipy # Installing 'runipy' for extended abilities to execute the notebook
runipy IntelModin_GettingStarted.ipynb # Test 'Modin is faster than Pandas' case
MODIN_CPUS=1 runipy IntelModin_GettingStarted.ipynb # Test 'Modin is slower than Pandas' case
```

</details>

2. Run the script from the sample's directory and ensure that the last notebook's cell printed"[CODE_SAMPLE_COMPLETED_SUCCESFULLY]"

Would also like to get a review from Modin folks: @Garra1980 @YarShev @vnlitvinov

### CODE REVIEW

File: AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb [hunk 1]
```diff
@@ -47,6 +47,60 @@
...
+    "        \"You may visit Modin's optimization guide in order to learn more about such cases and how to fix them: \"\n",
+    "        \"\\nhttps://modin.readthedocs.io/en/latest/usage_guide/optimization_notes/index.html\\n\\n\"\n",
+    "        \"But first, verify that you're using the latest Modin version, also, try to use different executions, \"\n",
+    "        \"for basic usage we recommend non-experimental 'PandasOnRay'\\n\"\n",
+    "        \"Current execution is:\"\n",
+    "    )\n",
+    "    try:\n",
+    "        from modin.utils import get_current_execution\n",
+    "\n",
+    "        execution = get_current_execution()\n",
```
Comment (dchigarev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb:75]: [The comment according `get_current_execution`](https://github.com/oneapi-src/oneAPI-samples/pull/843/files#r800373473) was applied

File: AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/sample.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
 			"conda activate intel-aikit-modin",
 			"pip install -r requirements.txt # Installing notebook's dependencies",
 			"pip install runipy # Installing 'runipy' for extended abilities to execute the notebook",
-			"runipy IntelModin_GettingStarted.ipynb"
+			"runipy IntelModin_GettingStarted.ipynb # Test 'Modin is faster than Pandas' case",
+			"MODIN_CPUS=1 runipy IntelModin_GettingStarted.ipynb # Test 'Modin is slower than Pandas' case"
```
Comment (dchigarev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/sample.json:23]: allowing for Modin to utilize only a single core to syntactically slow it down

Comment (YarShev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/sample.json:23]:
```suggestion
			"runipy IntelModin_GettingStarted.ipynb # Test 'Modin is faster than pandas' case",
			"MODIN_CPUS=1 runipy IntelModin_GettingStarted.ipynb # Test 'Modin is slower than pandas' case"
```

File: AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb [hunk 3]
```diff
@@ -47,6 +47,60 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ****** Do not change the code in this cell! It verifies that the notebook is being run correctly! ******\n",
+    "\n",
+    "def verify_and_print_times(pandas_time, modin_time):\n",
+    "    if modin_time < pandas_time:\n",
+    "        print(f\"Modin was {pandas_time / modin_time:.2f}X faster than stock Pandas!\")\n",
```
Comment (YarShev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb:60]:
```suggestion
    "        print(f\"Modin was {pandas_time / modin_time:.2f}X faster than stock pandas!\")\n",
```

File: AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb [hunk 4]
```diff
@@ -47,6 +47,60 @@
...
+    "        return\n",
+    "    print(\n",
+    "        f\"Oops, stock Pandas appears to be {modin_time / pandas_time:.2f}X faster than Modin in this case. \"\n",
```
Comment (YarShev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb:63]:
```suggestion
    "        f\"Oops, stock pandas appears to be {modin_time / pandas_time:.2f}X faster than Modin in this case. \"\n",
```

File: AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb [hunk 5]
```diff
@@ -47,6 +47,60 @@
...
+    "        \"This is unlikely but could happen sometimes on certain machines/environments/datasets. \"\n",
+    "        \"One of the most probable reasons is the excessive amount of partitions being assigned to a single worker. \"\n",
+    "        \"You may visit Modin's optimization guide in order to learn more about such cases and how to fix them: \"\n",
+    "        \"\\nhttps://modin.readthedocs.io/en/latest/usage_guide/optimization_notes/index.html\\n\\n\"\n",
+    "        \"But first, verify that you're using the latest Modin version, also, try to use different executions, \"\n",
+    "        \"for basic usage we recommend non-experimental 'PandasOnRay'\\n\"\n",
```
Comment (YarShev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb:69]:
```suggestion
    "        \"for basic usage we recommend non-experimental 'PandasOnRay'.\\n\"\n",
```

File: AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb [hunk 6]
```diff
@@ -47,6 +47,60 @@
...
+    "        try:\n",
+    "            from modin.utils import get_current_backend\n",
+    "\n",
+    "            execution = get_current_backend()\n",
+    "        except ImportError:\n",
+    "            # for modin versions < 0.8.1\n",
+    "            execution = (\n",
+    "                \"Can't deduce the current execution, your Modin version is too old!\"\n",
+    "            )\n",
+    "    print(f\"\\tExecution: {execution}\")\n",
```
Comment (YarShev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb:87]: We print above `Current execution is:`. Should we duplicate this?

File: AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb [hunk 7]
```diff
@@ -47,6 +47,60 @@
...
+    "    try:\n",
+    "        import modin.config as cfg\n",
+    "\n",
+    "        print(\n",
+    "            f\"\\tIs experimental: {cfg.IsExperimental.get()}\\n\"\n",
+    "            f\"\\tCores to utilize by Modin (check that Modin uses all cores on your machine): {cfg.CpuCount.get()}\\n\"\n",
```
Comment (YarShev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb:93]:
```suggestion
    "            f\"\\tNumber of CPUs to utilize by Modin (check that Modin uses all CPUs on your machine): {cfg.CpuCount.get()}\\n\"\n",
```

**Review by praveenkk123 — APPROVED** [review]
Approve for CI

**Comment by praveenkk123** [comment]
Can we merge this PR..thanks

**Review by praveenkk123 — APPROVED** [review]

**Pull request merged**